configs/edison/config.20161006bugfix.alpha8.A_WCYCL1850S.ne30_oEC_ICG.edison

```diff
@@ -131,6 +131,7 @@ polarPlot = False
 [streamfunctionMOC]
 ## options related to plotting the streamfunction of the meridional overturning 
 ## circulation (MOC)
+maxChunkSize = 1000
```

Comment:
To be honest, I'm not sure 100% because it depends on our ultimate goals.  A naive, simplest approach is to do chunking in the preprocessor of `open_mfdataset`, but I'm trying to figure out if I've missed something on why we have this problem.  At the 60to30 scale we can completely store the mean climatology in memory.  This is why going to numpy gave us good performance because there is the potential for a dask soft bug otherwise (https://github.com/dask/dask/issues/874).  

The real problem is that we take a huge performance hit computing the climatology, especially over a very long dataset.  The fundamental issue is that to compute the climatology requires reading in a lot of data off disk, which is the most expensive thing we do.  Assuming we do this in a way that completes, this will always be something very expensive.  I think we will need to specify a chunk size for this operation to make sure it doesn't exceed memory.  I just need to go back and make sure chunking happens in the preprocessor like we did in the previous PR.  We can ignore chunking for non-memory sensitive cases so that we get better performance (e.g., the whole spatial slices for each dataset file get loaded at once).  

Does this sound ok?  This is a short term solution.  Having multi-time scale climatology files computed and cached incrementally is the correct way for us to handle this type of problem in general.  Optimally, we do this all online with MPAS AMs.